internal/pkg/levee/levee_test.go

```diff
@@ -29,6 +29,9 @@ func TestLevee(t *testing.T) {
 	if err := Analyzer.Flags.Set("config", dataDir+"/test-config.yaml"); err != nil {
 		t.Error(err)
 	}
+	if err := Analyzer.Flags.Set("useEAR", "false"); err != nil {
```

Comment:
The flag is `false` by default, so this may not be necessary? Same comment for other tests in this file.

Reply:
The previous commit fails the tests in "levee_test.go". When "levee_ear_test.go" is run first, this flag is set to "true" there, and when the subsequent running of "levee_test.go" will fail. This is because "Analyzer.Flags" is a global.

Reply:
Ah, I was wondering if there might be some issue involving test ordering. Thanks, resolved.

Reply:
For testing, an "easier" answer is to just have different analysis.Analyzer objects for testing. Sure there is a bit of a disconnect from the "production" object, but this should not be a big deal if there is enough separation between the modes.

Reply:
Now the flags have been moved to `config.yaml`, and specific yaml files will be used for EAR. There are no globals for these flags anymore.